.devcontainer/Dockerfile

```diff
@@ -69,11 +69,11 @@ RUN echo "Install general purpose packages" && \
         tzdata \
         virtualenv=20.0.17-1ubuntu0.4 && \
     gem install fpm && \
-    update-alternatives --install /usr/bin/clang clang /usr/lib/llvm-11/bin/clang 10 && \
-    update-alternatives --install /usr/bin/clang++ clang++ /usr/lib/llvm-11/bin/clang++ 10 && \
-    update-alternatives --install /usr/bin/clang-format clang-format /usr/lib/llvm-11/bin/clang-format 10 && \
-    update-alternatives --install /usr/bin/clang-tidy clang-tidy /usr/lib/llvm-11/bin/clang-tidy 10 && \
-    update-alternatives --install /usr/bin/clang-apply-replacements clang-apply-replacements /usr/lib/llvm-11/bin/clang-apply-replacements 10
+    update-alternatives --install /usr/bin/clang clang /usr/lib/llvm-12/bin/clang 10 && \
+    update-alternatives --install /usr/bin/clang++ clang++ /usr/lib/llvm-12/bin/clang++ 10 && \
+    update-alternatives --install /usr/bin/clang-format clang-format /usr/lib/llvm-12/bin/clang-format 10 && \
+    update-alternatives --install /usr/bin/clang-tidy clang-tidy /usr/lib/llvm-12/bin/clang-tidy 10 && \
+    update-alternatives --install /usr/bin/clang-apply-replacements clang-apply-replacements /usr/lib/llvm-12/bin/clang-apply-replacements 10
```

Comment:
Do we need this, or could it be removed? Was that not only for compatibility with an additional old llvm installed?

Reply:
I think, we can remove it.